rsocket-transport-netty/src/main/java/io/rsocket/transport/netty/client/WebsocketClientTransport.java

```diff
@@ -130,6 +132,8 @@ public static WebsocketClientTransport create(HttpClient client, String path) {
     Objects.requireNonNull(client, "client must not be null");
     Objects.requireNonNull(path, "path must not be null");
 
+    path = path.startsWith(DEFAULT_PATH) ? path : (DEFAULT_PATH + path);
```

Comment:
Checking for "" seems safer than handling all non "/.*" cases.  Is it deliberate to convert something like ":443" to "/:443"

Reply:
guess it is better to check this on regex for allowed symbols. Since it is possible to do something like "test" for the `path` field, which also must start with "/"

Reply:
In general, path allows any symbols, so it does not matter. `URI#getPath` returns everything after "/" so if this is ":443" then, it is, probably, the right path. On the other hand, if it will be ":443", netty consider that as a `baseURL` and after some modification try to validate it against 
```
(?:(\w+)://)?((?:\[.+?])|(?<!\[)(?:[^/?]+?))(?::(\d{2,5}))?([/?].*)?
```

which fails validation.

See -> https://github.com/reactor/reactor-netty/blob/master/src/main/java/reactor/netty/http/client/HttpClient.java#L106
and -> https://github.com/reactor/reactor-netty/blob/master/src/main/java/reactor/netty/http/client/HttpClientConnect.java#L482

Reply:
Thus, it is safe for any of possible cases